### ISSUE TITLE
be able to turn off ellipsis 

### ISSUE THREAD

**Issue by gabrielcsapo** [rwjblue/broccoli-slow-trees#11]
👋 this is an awesome project

I was wondering if you would be up to add a flag that would allow you to turn off ellipsis?

I have a project that outputs:

```
Slowest Nodes (totalTime => 5% )              | Total (avg)         
----------------------------------------------+---------------------
broccoli-persistent-filter:TemplateC... (645) | 80985ms (125 ms)  
```

It would be great to turn this off so I can figure out what specifically is taking (1 minute 21 seconds)

**Comment by rwjblue** [comment]
The idea here is just to be a quick summary, if you want to dig into details you are going to have a much better time using `BROCCOLI_VIZ=1 ember s` and leveraging the heimdall visualizer.

**Comment by gabrielcsapo** [comment]
Right, my thoughts are most people will not know what to look for in the viz given that you only have a portion of the funnel name. This specific funnel is named the name for multiple addons.

**Comment by gabrielcsapo** [comment]
Thoughts on this use case?
